annotation_pipeline/populate_labelstudio.py

```diff
@@ -0,0 +1,243 @@
+from huggingface_hub import hf_hub_download
```

Comment:
To make things extra super-duper clear, let's put a docstring at the top of this, so that anyone viewing the file knows right away what this file is and what it does.